content/en/docs/rules/basic-elements.md

```diff
@@ -250,3 +250,12 @@ In other words, visibility is defined in cascade and is quite important:
 - A macro can only reference macros defined before it.
 - A macro can reference any list.
 - A rule can reference any macro.
+
+
+## Highly Relevant Blogposts and Resources
+
+- [Adaptive Syscalls Selection in Falco](https://falco.org/blog/adaptive-syscalls-selection/).
+- [Validating NIST Requirements with Falco](https://falco.org/blog/falco-nist-controls/).  
+- [PCI/DSS Controls with Falco](https://falco.org/blog/falco-pci-controls/).
+- [Style Guide of Falco Rules](/docs/contribute/style-guide/).
+- [Contributing](/docs/contribute/) page and [How to Share Your Falco Rules](/docs/contribute/share-rules/) guide.
```

Comment:
Are we sure this is the best place to introduce such advanced content? IMHO, most users would land there to learn the basics about rules, not to use them in such an advanced way.

Reply:
Found a new place WDYT?